boards/arm/mimxrt685_evk/mimxrt685_evk_cm33.dts

```diff
@@ -70,17 +70,17 @@
 	pwmleds {
 		compatible = "pwm-leds";
 		green_pwm_led: green_pwm_led {
-			pwms = <&sc_timer 0 PWM_POLARITY_NORMAL>;
+			pwms = <&sc_timer 0 PWM_MSEC(20) PWM_POLARITY_NORMAL>;
 			label = "Green PWM LED";
 			status = "okay";
 		};
 		blue_pwm_led: blue_pwm_led {
-			pwms = <&sc_timer 6 PWM_POLARITY_NORMAL>;
+			pwms = <&sc_timer 6 PWM_MSEC(20) PWM_POLARITY_NORMAL>;
 			label = "Blue PWM LED";
 			status = "okay";
 		};
 		red_pwm_led: red_pwm_led {
-			pwms = <&sc_timer 6 PWM_POLARITY_NORMAL>;
+			pwms = <&sc_timer 6 PWM_MSEC(20) PWM_POLARITY_NORMAL>;
```

Comment:
This seems to be wrong - both these LEDs use the same channel. I only spotted it now and I actually have no idea whether/how this should be fixed.

Reply:
Hmm you're right. I'll check the board schematic and see if it can be fixed as part of this PR. Maybe @dleach02 or @danieldegrasse have some input here.

Reply:
I confirmed and this is how it is connected on the board